packages/scandipwa/src/component/PasswordChangeForm/PasswordChangeForm.form.js

```diff
@@ -51,7 +51,8 @@ export const customerEmailAndPasswordFields = (range, minimunPasswordCharacter)
             inputType: VALIDATION_INPUT_TYPE.password,
             match: (value) => {
                 const password = document.getElementById('password');
-                return password.value === value;
+                const password_confirmation = document.getElementById('password_confirmation');
```

Comment:
Using the `getElementById` method is a bad practice in the React. Could you refactor this and use refs, please?

Reply:
Please check the `elemRef` prop of the `Field` component.

Reply:
done